src/views/transaction-details/transaction-details.styles.ts

```diff
@@ -0,0 +1,116 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+import { TransactionCardProps } from "src/views/activity/components/transaction-card/transaction-card.view";
+
+interface StylesProps {
+  status: TransactionCardProps["status"];
+}
+
+const useTransactionDetailsStyles = createUseStyles((theme: Theme) => ({
+  card: {
+    padding: theme.spacing(3),
+  },
+  dot: {
+    width: theme.spacing(0.75),
+    height: theme.spacing(0.75),
+    backgroundColor: ({ status }: StylesProps) => {
+      switch (status) {
+        case "completed":
+          return theme.palette.success;
+        case "on-hold":
+        case "failed":
+          return theme.palette.error.main;
+        default:
+          return theme.palette.warning.main;
+      }
+    },
+    borderRadius: "50%",
+  },
+  balance: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    flexDirection: "column",
+    gap: theme.spacing(1),
+    borderBottom: `1px solid ${theme.palette.grey.light}`,
+    paddingBottom: theme.spacing(3),
+    marginBottom: theme.spacing(1.5),
+  },
+  tokenIcon: {
+    width: theme.spacing(6),
+    height: theme.spacing(6),
```

Comment:
Using theme spacing to define sizes will produce unexpected size changes when one decides to change the spacing.

Reply:
Fixed.